patches/0016-user-agent-set-to-Pulumi-apn-1.1-info.patch

```diff
@@ -0,0 +1,73 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: corymhall <43035978+corymhall@users.noreply.github.com>
```

Comment:
I would be happy change to me or add me as Co-Author (once I learn how). Thoughts?

Reply:
How did you generate the patch? I think it should have just picked up your git info

Reply:
I intended to replace the patch with the new code so we don't have two patches for APN setting.

Reply:
What I did was:

1. `./scripts/upstream.sh rebase -i`
2. Placed `edit` next to commit pointed byt `patches/0016-user-agent-set-to-Pulumi-apn...`
3. Added files and made edits.
4. Staged my changes and `git commit --amend`
5. `./scripts/upstream.sh check_in`

I believe those steps are equivalent with what you described on _What i've done in the past..._.